### PR TITLE
Implement fellowship admin dashboard 

### LINKED ISSUE #96: Create a view to add certificates
**Is your feature request related to a problem? Please describe.**
Need a new view to create new certificates.

**Describe the solution you'd like**
* Create a new view to with a one text input to enter the name for the certificate and save it as a certificate.
* Add Fellowship tab to the sider and the add menu on the navbar and nest this link as the add certificates.

**Additional context**
follow this [link](https://sef.discourse.group/t/developing-an-admin-platform-for-sef-fellowship-program/20) for more details.

### LINKED ISSUE #97: Create a view to list all the certificates
**Is your feature request related to a problem? Please describe.**
Need a view to list down all the certificates.

**Describe the solution you'd like**
* Use the same template as the academix category view and create a table to render all the certificates.
* Add a button to delete the certificate. 
* Add a button to copy the link to the certificate. It should generate in the following manner,
`sefglobal.org/fellowship/?id={ id of the certificate }`

**Additional context**
For more details follow [this link](https://sef.discourse.group/t/developing-an-admin-platform-for-sef-fellowship-program/20) to the hive thread.

### PR DESCRIPTION
**Pull request by Gravewalker666**
## Purpose

The purpose of this PR is to fix #97 fix #96 

## Goals

To develop the dashboard to control fellowship APIs

## Screenshots
image

## Approach

* Created two new scenes `Upsert Certificate` and `Certificates`
* Listed down all the certificates in the `Certificates` scene and added buttons to `edit`, `delete` and `copy the pubic link`
* Created a new endpoint to update certificates under `/core/fellowship/admin/certificates/id`
* Implemented `Upsert Certificate` component to handle both put and post requests

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Test environment
 
Java Version: OpenJDK 11.0.7
TestServer: Tomcat 9
Os: MacOs Catalina 10.15.3
Test: Google Chrome

## Learning

* https://www.npmjs.com/package/react-copy-to-clipboard

### CODE REVIEW

**Review by anjula-sack — APPROVED** [review]
Awesome! @Gravewalker666

File: backend/src/main/java/org/sefglobal/core/fellowship/controller/admin/CertificateAdminController.java [hunk 1]
```diff
@@ -34,4 +34,10 @@ public Certificate addCertificate(@RequestBody Certificate certificate){
     public String deleteCertificateById(@PathVariable Integer id) throws ResourceNotFoundException {
         return certificateService.deleteCertificate(id);
     }
+
+    @PutMapping("/{id}")
+    @ResponseStatus(HttpStatus.OK)
+    public Certificate updateCertificateById(@PathVariable Integer id, @RequestBody Certificate certificate) throws ResourceNotFoundException {
```
Comment (YohanAvishke) [inline backend/src/main/java/org/sefglobal/core/fellowship/controller/admin/CertificateAdminController.java:40]: Hi let's format this sentence it's too long.
```suggestion
    public Certificate updateCertificateById(@PathVariable Integer id, 
                                             @RequestBody Certificate certificate) 
            throws ResourceNotFoundException {
```

Comment (Gravewalker666) [reply backend/src/main/java/org/sefglobal/core/fellowship/controller/admin/CertificateAdminController.java]: Sure!

Comment (Gravewalker666) [reply backend/src/main/java/org/sefglobal/core/fellowship/controller/admin/CertificateAdminController.java]: Resolved with commit [24d10be](https://github.com/sef-global/sef-core/pull/99/commits/24d10be28040012d2f837a232501ddd5d7424d52)

**Review by jayasanka-sack — APPROVED** [review]
Thanks @Gravewalker666 !

**Pull request merged**